package/Distribution.xml

```diff
@@ -27,7 +27,7 @@
   <license file="LICENSE.rtf"/>
   <conclusion file="CONCLUSION.rtf" />
   <allowed-os-versions>
-    <os-version min="11.0"/>
+    <os-version min="12.0.0"/>
```

Comment:
I'd really like to avoid a `.in` flow here for a number that's changed once a year.

I'm game for you to remove this if you verify `--min-os-version` is sufficient.